charts/alfresco-common/Chart.yaml

```
@@ -5,5 +5,6 @@ description: |
   A helper subchart to avoid duplication in alfresco charts and set common
   external dependencies
 type: library
-version: 4.0.0-alpha.0
+version: 4.0.0-alpha.1
+dependencies: []
```

```suggestion
version: 4.0.0-alpha.0
```

the release didn't happen so no bump to do?

just run `helm dep up` in the alfresco-common folder